### ISSUE TITLE
Organize crates

### ISSUE THREAD

**Issue by dabreegster** [a-b-street/abstreet#844]
We have lots of them now, and the flattened directory listing is a little overwhelming? Here's a vague attempt at organizing, not happy with it yet.

apps
	fifteen_min
	game
	ltn
	map_editor
	osm_viewer
	parking_mapper
	santa

project / logistic
	cloud
	release
	tests
	traffic_seitan
	web

importing
	importer
	collisions
	convert_osm
	kml
	traffic_signal_data
	popdat

ui
	map_gui
	piggyback
	widgetry
	widgetry_demo

(uncategorized)

abstio
abstutil
cli
geom
headless
map_model
sim
updater

**Comment by dabreegster** [comment]
https://matklad.github.io/2021/08/22/large-rust-workspaces.html - "Flat Is Better Than Nested"

apps/ was an obvious one, but not sure this is worth doing further. Grouping map model things -- maybe? Where's the line between map and sim? (Possibly synthpop). Do the importing tools belong in one or another?

**Comment by Robinlovelace** [comment]
Good shout to organise them but, without this being an informed opinion, wondering if it's more important to communicate the structure in the docs than in the organisation of the repo?

**Comment by dabreegster** [comment]
Yeah, updating https://a-b-street.github.io/docs/tech/dev/index.html#code-organization would be useful. I guess the dust has settled here enough for that now
